src/Entities/Player.h

```diff
@@ -590,11 +590,18 @@ class cPlayer:
 	The player removes its m_ClientHandle ownership so that the ClientHandle gets deleted. */
 	void RemoveClientHandle(void);
 
-	/** Returns the relative block hardness for the block a_Block.
-	The bigger it is the faster the player can break the block.
-	Returns zero if the block is instant breakable.
-	Otherwise it returns the dig speed (float GetDigSpeed(BLOCKTYPE a_Block)) divided by the block hardness (cBlockInfo::GetHardness(BLOCKTYPE a_Block)) divided by 30 if the player can harvest the block and divided by 100 if he can't. */
-	float GetPlayerRelativeBlockHardness(BLOCKTYPE a_Block);
+	/** Returns the progress mined per tick for the block a_Block as a fraction
+	(1 would be completely mined)
+	Returns 0 if it can be instantly mined.
```

Comment:
Wouldn't returning 1 make more sense?

Reply:
It really would, if I had a reason for that inconsistency I've since forgotten, thanks!